CHANGELOG.rst

```diff
@@ -75,6 +75,9 @@ Fixed
 * Fixed a regression in the ``linux.dig`` action on Python 3. (bug fix) #4993
 
   Contributed by @blag
+* Fix actions from the contrib/linux pack that fail on CentOS-8 but work on other operating systems and distributions.
+
+  Reported by @blag and contributed by winem (bug fix) #4999
```

Comment:
A very small adjustment to the Changelog format:

```suggestion
* Fix actions from the contrib/linux pack that fail on CentOS-8 but work on other operating systems and distributions.  (bug fix) #4999

  Reported by @blag and contributed by @winem
```

Reply:
I actually had #4999 in mind when I wrote the changelog. But I don't mind to mention @dove-young as well although I don't tackle the issue with the linux.service action here.

Reply:
I updated the changelog and @dove-young is mentioned now, too. Using the st2 py fixes the issue with the linux.service action as well. So all actions in the linux pack will be fixed with this PR now.

Reply:
Thanks! I just was confused by the #5004 reference. Looks clear now :+1: